packages/application-shell/src/components/application-shell-provider/application-shell-provider.tsx

```diff
@@ -82,7 +81,5 @@ const ApplicationShellProvider = (props: TApplicationShellProviderProps) => {
 };
 
 ApplicationShellProvider.displayName = 'ApplicationShellProvider';
-// This is useful to inject a custom history object during tests
-ApplicationShellProvider.history = history;
```

Comment:
Refactored this workaround (as TS was complaining) to mock the util function `getBrowserHistory`.